pkg/utils/k8sutil/pravega.go

```diff
@@ -41,11 +49,11 @@ func DeploymentNameForController(clusterName string) string {
 }
 
 func ConfigMapNameForSegmentstore(clusterName string) string {
-	return fmt.Sprintf("%s-segmentstore", clusterName)
+	return fmt.Sprintf("%s-pravega-segmentstore", clusterName)
```

Comment:
Given that the cluster name is the prefix, and will have the value `pravega` in most cases, it seems flat wrong to add `-pravega-`. What does it accomplish?

Reply:
The cluster name is user defined, so it might or might not contain "pravega". I've added the `- pravega` prefix to be consistent with other components that already use the same prefix and to be more verbose and indicate that it's a Pravega resource.